<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Lenovo" failures="0" tests="2" name="com.time_out.Time_Out_Ex" time="4.110" errors="1" timestamp="2022-08-03T12:22:04 IST" skipped="0">
  <testcase classname="com.time_out.Time_Out_Ex" name="amazon" time="3.051"/>
  <testcase classname="com.time_out.Time_Out_Ex" name="flipkart" time="1.059">
    <error message="Method com.time_out.Time_Out_Ex.flipkart() didn&#039;t finish within the time-out 1000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.time_out.Time_Out_Ex.flipkart() didn't finish within the time-out 1000
]]>
    </error>
  </testcase> <!-- flipkart -->
</testsuite> <!-- com.time_out.Time_Out_Ex -->
